tests/main/refresh-devmode/task.yaml

```diff
@@ -16,8 +16,14 @@ environment:
     STORE_TYPE/remote: ${REMOTE_STORE}
 
 prepare: |
-    if [[ "$STORE_TYPE" = "fake" ]] && [[ "$SPREAD_SYSTEM" =~ ubuntu-core-16-* ]]; then
-        exit
+    if [ "$STORE_TYPE" = "fake" ]; then
+        if [[ "$SPREAD_SYSTEM" =~ ubuntu-core-16-* ]]; then
```

Comment:
this conditional is buggy. I know it was that way before as well, but still. With the `=~` operator the string on the right is a regular expression, meaning that it's looking for `ubuntu-core-16` followed by an arbitrary number of `-`s.

What it's wanting to match is, I suspect, `^ubuntu-core-16-.*$` (note that without the `^` and `$` anchoring it will match anywhere in the string, not the whole string), or alternatively use the `==` operator and then the right hand side is a pattern that can have globs.